internal/db/dbtest/dbtest.go

```diff
@@ -0,0 +1,26 @@
+package dbtest
+
+import (
+	"net/http"
+	"testing"
+	"wallet-backend/internal/db/migrations"
+
+	migrate "github.com/rubenv/sql-migrate"
+	"github.com/stellar/go/support/db/dbtest"
+	"github.com/stellar/go/support/db/schema"
+)
+
+func Open(t *testing.T) *dbtest.DB {
+	db := dbtest.Postgres(t)
+	conn := db.Open()
+	defer conn.Close()
+
+	migrateDirection := schema.MigrateUp
+	m := migrate.HttpFileSystemMigrationSource{FileSystem: http.FS(migrations.FS)}
```

Comment:
❓ Can you explain how we are going to use this? Is it not going to use files in a certain directory?

Reply:
It is, `migrations.FS` is a reference to the file system (directory) where the migrations live. It is defined in `migrations/main.go`. It becomes an absolute reference so that we no longer need to worry about providing relative paths whenever opening the DB connection. Even if the migrations folder moves, the reference won't be broken.